apps/www/registry/default/ui/card.tsx

```diff
@@ -2,78 +2,60 @@ import * as React from "react"
 
 import { cn } from "@/lib/utils"
 
-const Card = React.forwardRef<
-  HTMLDivElement,
-  React.HTMLAttributes<HTMLDivElement>
->(({ className, ...props }, ref) => (
+const Card: React.FC<React.ComponentProps<"div">> = ({
+  className,
+  ...props
+}) => (
   <div
-    ref={ref}
     className={cn(
-      "rounded-lg border bg-card text-card-foreground shadow-sm",
+      "bg-card text-card-foreground rounded-lg border shadow-sm",
       className
     )}
     {...props}
   />
-))
+)
 Card.displayName = "Card"
 
-const CardHeader = React.forwardRef<
-  HTMLDivElement,
-  React.HTMLAttributes<HTMLDivElement>
->(({ className, ...props }, ref) => (
-  <div
-    ref={ref}
-    className={cn("flex flex-col space-y-1.5 p-6", className)}
-    {...props}
-  />
-))
+const CardHeader: React.FC<React.ComponentProps<"div">> = ({
+  className,
+  ...props
+}) => (
+  <div className={cn("flex flex-col space-y-1.5 p-6", className)} {...props} />
+)
 CardHeader.displayName = "CardHeader"
 
-const CardTitle = React.forwardRef<
-  HTMLParagraphElement,
-  React.HTMLAttributes<HTMLHeadingElement>
->(({ className, ...props }, ref) => (
+const CardTitle: React.FC<React.ComponentProps<"h3">> = ({
+  className,
+  ...props
+}) => (
   <h3
```

Comment:
This is not the only component that was originally using a paragraph element for headings. To be fair, it doesn't make THAT huge of a difference, but it was gonna bother me if I didn't fix it 😅